qp_klp/CHECKLIST.txt

```diff
@@ -0,0 +1,9 @@
+Confirm sif behavior works w/new KATHAROseq controls.
```

Comment:
Intentionally added. A todo list stored in the PR to make sure I wouldn't lose it. Since we squash before merge I figured it was okay to do this as long as I removed it beforehand.

Reply:
Got it, can you remove it?